src/config/constants.js

```diff
@@ -0,0 +1,48 @@
+import merge from 'lodash/merge';
+import packageJson from '../../package.json';
+
+// Default configurations applied to all environments
+const defaultConfigs = {
+    env: process.env.ENV || 'dev',
+    version: packageJson.version,
+};
+
+// Environment specific overrides
+const environmentConfigs = {
+    dev: {
+        cognito: {
+            userPoolId: process.env.USER_POOL_ID || 'us-east-1_HetaxzWw5',
+            ISS: process.env.USER_POOL_ID || 'https://cognito-idp.us-east-1.amazonaws.com/us-east-1_HetaxzWw5/',
+        },
+        db: {
+            host: process.env.USER_POOL_ID || '52.15.81.107',
+            user: process.env.USER_POOL_ID || 'root',
+            password: process.env.USER_POOL_ID || 'password',
```

Comment:
@SavidyaSathyani  @singhabahu These configs should be reflected in the relevant areas where we used, ryt ?

Reply:
If we are to keep this constant file, then yes we need to access configs  through this file in our code base. I didn't include that implementation since this is only to add the feature. Future enhancements must be done to the code if this gets approved.